### DISCUSSION TITLE
Ability to Rebind Shortcuts

### DISCUSSION THREAD

**Discussion by p-romeo** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#508] [Ideas]
Add option to rebind opening shortcut and to close assistant with a key (namely escape)

**Comment by Melvin-Abraham** [comment]
This has been implemented in a4d21cb5c8 but not released yet. Until the next release is up, you can test the feature out by [building it on your machine](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build) if you are curious :)
